admin/page/berita.php

```diff
@@ -0,0 +1,115 @@
+<script src="https://cdn.ckeditor.com/ckeditor5/19.1.1/decoupled-document/ckeditor.js"></script>
+
+
+<div class="mt-4 w-100">
+    <input type="search" placeholder="Search" id="search" name="search" class="search ml-auto float-right">
+</div>
+
+<table class="table" id="tabel">
+    <thead>
+        <th style="background-color: #204a87ff; text-align: center;">ID</th>
+        <th style="background-color: #204a87ff; text-align: center;">Judul Berita</th>
+        <th style="background-color: #204a87ff; text-align: center;">Tanggal Berita</th>
+        <th style="background-color: #204a87ff; text-align: center;">Isi Berita</th>
+        <th style="background-color: #204a87ff; text-align: center;">Gambar Berita</th>
+        <th style="background-color: #204a87ff; text-align: center;">Kategori Berita</th>
+    </thead>
+    <tbody id="tbody">
+        <?php
+            $select = mysqli_query($connection, "select * from berita");
+            while($data = mysqli_fetch_array($select)){
+                echo "
+                    <tr data-toggle='modal' data-target='#update-berita'>
+                        <td class='text-center'>".$data["id_berita"]."</td>
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$data["id_berita"]'.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)